pkg/k8s/k8s.go

```diff
@@ -1,232 +0,0 @@
-package k8s
```

Comment:
Why has this entire file been removed? A lot of the stuff inside is used in `viz`

Reply:
Thanks for catching this, I didn't know how I could mistake that. Obviously kubectl.go was supposed to be deleted. It was only used in outdated healthcheck.